mobile-app/app/components/BottomSheetWithNav.tsx

```diff
@@ -75,7 +75,11 @@ export const BottomSheetWithNav = React.memo((props: BottomSheetWithNavProps): J
   )
 })
 
-export const BottomSheetWebWithNav = React.memo((props: BottomSheetWithNavProps & { isModalDisplayed: boolean, modalStyle?: { [other: string]: any} }): JSX.Element => {
+const EmptyHandleComponent = (_props: BottomSheetHandleProps): JSX.Element => {
```

Comment:
can we use props instead of having underscore

Reply:
removed, since it's unused